installation/resources/installer-config-cluster.yaml.tpl

```diff
@@ -126,4 +126,5 @@ metadata:
     component: knative
 data:
   knative.ingressgateway.service.type: LoadBalancer
-  knative.domainName: "__DOMAIN__"
\ No newline at end of file
+  knative.domainName: "__DOMAIN__"
+  
```

Comment:
Please add a new line

Reply:
@radufa The file now ends with a newline, this is a good practice :)